src/Layout/reduxer.js

```diff
@@ -41,6 +41,7 @@ export const ReduxTopBar = (() => {
   const mapStateToProps = state => ({
     navTitle: components.navigation.title,
     topItems: components.topBar.items,
+    useProfileDropdown: components.topBar.useProfileDropdown,
```

Comment:
can you update `/docs/portal_config.md` about this new option as well?

Reply:
Done